packages/fullstack-protocol/src/lib.rs

```diff
@@ -88,7 +92,8 @@ impl HydrationContext {
         &self,
         id: usize,
         value: &T,
-        location: &'static std::panic::Location<'static>,
+        #[cfg(debug_assertions)] location: &'static std::panic::Location<'static>,
+        #[cfg(not(debug_assertions))] _location: &'static std::panic::Location<'static>,
     ) {
         self.data.borrow_mut().insert(id, value, location);
```

Comment:
This will not compile in release mode because the location variable is not defined